.github/workflows/cargo.yml

```diff
@@ -0,0 +1,11 @@
+name: cargo
```

Comment:
I named it this way so that the action summary matches the command we're running.

![image](https://github.com/user-attachments/assets/1c500b08-5f4e-435d-afd1-a09e3f5919fd)